doc/man3/OSSL_PROVIDER.pod

```diff
@@ -47,6 +52,9 @@ OSSL_PROVIDER_unload() unloads the given provider.
 For a provider added with OSSL_PROVIDER_add_builtin(), this simply
 runs its teardown function.
 
+OSSL_PROVIDER_available() checks if a named provider is available
+for use.
```

Comment:
Should we state here about the behaviour wrt fallbacks?

Reply:
No, that's internal